docs/react-v9/contributing/unstable-hooks.md

```diff
@@ -0,0 +1,15 @@
+# How to treat API's marked with the `_unstable` suffix
+
+There are many hooks and functions throughout this library marked with the `_unstable` suffix.
+
+This is due to legacy decisions in the early days of v9 development, when there was a chance these API's might change.
+
+Since then, the library has continued to move and grow. We've learned stability was something partners treasured, so we've also embraced that value. It was concluded that these API's should not be changed, even to rename them in the v9 iteration of Fluent UI React.
+
+Renaming an API to remove the suffix would be a breaking change. Marking the `_unstable` API's as `@deprecated`with JSDocs and re-exporting them would also trip many teams linters and cause friction for consuming teams.
+
+**We treat API's with the `_unstable` suffix as stable, and our consumers should do also consider them stable. This should be considered simply a naming bug at this point.**
```

Comment:
I think we should not that this is only true for stable packages and document how to know if a package is stable. Off the top of my head a stable package:

1. Is under "Components" on react.fluentui.dev
2. Has a 9.x.y version number
3. Is exported from `@fluentui/react-components`.

Conversely, unstable packages:

1. Are under "Preview Components" on react.fluentui.dev
2. Have a 0.x.y version number
3. Have a `-preview` suffix on the package name
4. Are never exported from `@fluentui/react-components`.

Reply:
yeah, it's part of https://github.com/microsoft/fluentui/blob/master/docs/react-v9/contributing/new-release-process-v9-packages.md

Reply:
Good call out - thanks Sean!